opus/application/static_media/js/browse.js

```diff
@@ -624,12 +626,26 @@ var o_browse = {
 
     // find the first displayed observation index & id in the upper left corner
     updateSliderHandle: function() {
+        let tab = `#${opus.prefs.view}`;
+        let contentsView = o_browse.getScrollContainerClass();
         let selector = (opus.prefs.browse === "dataTable") ? `#${opus.prefs.view} #dataTable tbody tr` : `#${opus.prefs.view} .gallery .thumbnail-container`;
+
+        // Fot gallery view, the topBoxBoundary is the top of .gallery-contents
+        // For table view, we will set the topBoxBoundary to be the bottom of thead (account for height of thead)
+        let topBoxBoundary = (opus.prefs.browse === "dataTable") ? $(".gallery-contents").offset().top + $(`${tab} #dataTable thead th`).outerHeight() : $(".gallery-contents").offset().top;
+
+        let currentObsNum = $(`${tab} ${contentsView}`).data("infiniteScroll").options.obsNum;
```

Comment:
I assume the infinite scroll object has been instantiated by this time so you will not throw an exception...?